share/picongpu/tests/Synchrotron_plugin/bin/ci.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+#
+# This file is part of PIConGPU.
+# Copyright 2023 PIConGPU contributors
+# Authors: Mika Soren Voss, Hannes Wolf, Klaus Steiniger, Filip Optolowicz
```

Comment:
not sure if all other names must be on it. Keep the names if you copied most of the code else remove other names.

Reply:
Almost all of the code is the same. I'm keeping it